app/components/typography/NumberBadge.tsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+interface NumberBadgeProps {
+  count: number;
+}
+
+const NumberBadge: React.FC<NumberBadgeProps> = ({ count }) => {
```

Comment:
While count is required in the interface, it’s good practice to ensure the component doesn’t break if the value is undefined (e.g., due to user error or edge cases). Consider adding a fallback value, such as 0.
```suggestion
const NumberBadge: React.FC<NumberBadgeProps> = ({ count = 0 }) => {
```

Reply:
done